packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -149,6 +150,92 @@ export class ProductCard extends PureComponent {
         );
     }
 
+    renderBundlePriceBadge(bounds) {
+        const {
+            product: { type_id, price_view }
+        } = this.props;
+
+        if (type_id !== BUNDLE) {
+            return null;
+        }
+
+        // eslint-disable-next-line
+        let bundleBadgeLabel;
+
+        if (bounds === 'bottom') {
+            bundleBadgeLabel = 'From';
+
+            if (price_view !== 'PRICE_RANGE') {
+                bundleBadgeLabel = 'As low as';
+            }
+        }
+
+        if (bounds === 'top') {
+            bundleBadgeLabel = 'To';
+        }
+
+        return (
+            <p
+              mix={ {
+                  block: 'ProductCard',
+                  elem: 'PriceBadge'
+              } }
+            >
+                { __(bundleBadgeLabel) }
```

Comment:
Please don't put variables in the translation function, since strings are being parsed from it.
Thus translatable string should be constant.
```javascript
// this is bad
const label = 'label';
return __(label);

// this is good
const label = __('label');
return label;
```